specification/paths/Payments.json

```diff
@@ -0,0 +1,162 @@
+                        ]
+                      },
+                      {
+                        "properties": {
+                          "relationships": {
+                            "required": [
+                              "organization",
+                              "status"
+                            ]
+                          }
+                        }
+                      }
+                    ]
+                  }
+                },
+                "meta": {
+                  "$ref": "#/components/schemas/PaginationMeta"
+                },
+                "links": {
+                  "type": "object",
+                  "required": [
+                    "self",
+                    "first",
+                    "last"
+                  ],
+                  "additionalProperties": false,
+                  "properties": {
+                    "self": {
+                      "type": "string",
+                      "example": "$API_HOST/payments?page[number]=3&page[size]=30"
+                    },
+                    "first": {
+                      "type": "string",
+                      "example": "$API_HOST/payments?page[number]=1&page[size]=30"
+                    },
+                    "prev": {
+                      "type": "string",
+                      "example": "$API_HOST/payments?page[number]=2&page[size]=30"
+                    },
+                    "next": {
+                      "type": "string",
+                      "example": "$API_HOST/payments?page[number]=4&page[size]=30"
+                    },
+                    "last": {
+                      "type": "string",
+                      "example": "$API_HOST/payments?page[number]=13&page[size]=30"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "post": {
+    "tags": [
+      "Payments"
+    ],
+    "security": [{
+      "OAuth2": []
+    }],
+    "summary": "Create an payment.",
+    "description": "This endpoint creates a new payment with the posted data.",
+    "requestBody": {
+      "description": "The payment object to be created.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "data": {
+                "allOf": [
+                  {
+                    "$ref": "#/components/schemas/Payment"
+                  },
+                  {
+                    "required": [
+                      "attributes"
+                    ]
+                  },
+                  {
+                    "properties": {
+                      "relationships": {
+                        "required": [
+                          "organization",
+                          "status"
```

Comment:
I think (but I'm not certain) status can be removed as this can always be set to a default status when posting.